### PR TITLE
Add benchmarks results with new GaussianBlur implementation

### PR DESCRIPTION
**Pull request by Ra1Nik**
Add benchmarks results with the new GaussianBlur on imagenet100 and imagenette. Cifar10 is not affected by gaussian blur.

### CODE REVIEW

**Comment by guarin** [comment]
Did some extra cleanup for the docs.

Also noticed that the CosineWarmupScheduler was not correctly tested and had a bug. LR went from 0 to 1 instead of 1 to 0.

File: examples/pytorch_lightning/vicregl.py [hunk 1]
```diff
@@ -37,8 +37,8 @@ def forward(self, x):
 
     def training_step(self, batch, batch_index):
         (view_global, view_local, grid_global, grid_local), _, _ = batch
-        z_global, z_global_local_features = model(view_global)
-        z_local, z_local_local_features = model(view_local)
+        z_global, z_global_local_features = self(view_global)
```
Comment (philippmwirth) [inline examples/pytorch_lightning/vicregl.py:40]:
```suggestion
        z_global, z_global_local_features = self.forward(view_global)
        z_local, z_local_local_features = self.forward(view_local)
```
to make it more explicit

File: examples/pytorch_lightning_distributed/vicregl.py [hunk 2]
```diff
@@ -36,8 +36,8 @@ def forward(self, x):
     
     def training_step(self, batch, batch_index):
         (view_global, view_local, grid_global, grid_local), _, _ = batch
-        z_global, z_global_local_features = model(view_global)
-        z_local, z_local_local_features = model(view_local)
+        z_global, z_global_local_features = self(view_global)
```
Comment (philippmwirth) [inline examples/pytorch_lightning_distributed/vicregl.py:39]: Same change as suggested above.

File: lightly/utils/scheduler.py [hunk 3]
```diff
@@ -91,8 +100,8 @@ def scale_lr(self, epoch: int) -> float:
 
         """
         if epoch < self.warmup_epochs:
-            return epoch / self.warmup_epochs
+            return (epoch + 1) / self.warmup_epochs
         else:
             return cosine_schedule(
-                epoch - self.warmup_epochs, self.max_epochs - self.warmup_epochs, 0, 1
+                epoch - self.warmup_epochs, self.max_epochs - self.warmup_epochs, 1.0, 0.0
```
Comment (philippmwirth) [inline lightly/utils/scheduler.py:106]: Please use keyword arguments here to prevent similar errors from happening

**Review by philippmwirth — CHANGES_REQUESTED** [review]
Looks great, just a few smaller suggestions.

**Review by philippmwirth — APPROVED** [review]

**Pull request merged**